expansionTest.file.spec.js

```diff
@@ -0,0 +1,20 @@
+it('visit GU website', () => {
```

Comment:
Line 1 doesn't have an assertion. The final command is clicking 'expansion', not testing something specific.